README.md

```diff
@@ -78,36 +84,39 @@ Configuration file is "config.json" by default:
     }
   ],
   "RedirectIPv6Record": true,
-  "IPNetworkFile": "/xx/xx.txt",
-  "DomainFile": "/xx/xx.txt",
+  "IPNetworkFile": "/path/to/ip_network_file",
+  "DomainFile": "/path/to/domain_file",
   "DomainBase64Decode": true,
-  "HostsFile": "./hosts",
+  "HostsFile": "/path/to/hosts_file",
   "MinimumTTL": 0,
   "CacheSize" : 0
 }
 ```
 
 Tips:
 
-+ BindAddress: No IP means listen both IPv4 and IPv6, overture will listen both TCP and UDP ports.
-+ DNS: You can use multiple dns upstream in this array.
-    + Name: Just for log.
-    + Protocol: "tcp" or "udp".
-    + EDNSClientSubnet: Improve dns accuracy. [RFC7871](https://tools.ietf.org/html/rfc7871)
++ BindAddress: Specifying only port (e.g. `:53`) will have overture listen on all available addresses (both IPv4 and 
+IPv6). Overture will handle both TCP and UDP requests.
++ DNS: You can specify multiple DNS upstream servers here.
+    + Name: This field is only used for logging
+    + Protocol: `tcp` or `udp`
+    + EDNSClientSubnet: Used to improve DNS accuracy. Please check [RFC7871](https://tools.ietf.org/html/rfc7871) for 
+    details.
         + Policy: 
-            + auto: If client IP is not in the reserved ip network, use client IP. Otherwise, use external IP.
+            + auto: If client IP is not in the reserved IP network, use client IP. Otherwise, use external IP.
             + disable: Disable this feature.
-        + ExternalIP: If this field is empty, edns client subnet will be disabled when use it.
-+ RedirectIPv6Record: Redirect IPv6 dns query to alternative dns.
-+ File: For windows user, if you want to use absolute path, please try like this: "C:\\\xx\\\xx.txt".
-+ DomainBase64Decode: If this file is base64 decoded, use "true".
-+ MinimumTTL: Set the minimum TTL value (second) in order to improve cache sufficiency, use 0 to disable.
+        + ExternalIP: If this field is empty, EDNS client subnet will be disabled when used.
++ RedirectIPv6Record: Redirect IPv6 DNS queries to alternative DNS servers.
++ File: For Windows users, you can use relative path like `.\file.txt`, or absolute path like `C:\path\to\file.txt` in
```

Comment:
It should be written as `C:\\path\\to\\file.txt`in json file. Markdown has my word escaped.........